src/xmlcli/common/tree_and_node.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file breaks[ PEP-8 naming convention guideline ](https://peps.python.org/pep-0008/#naming-conventions)